deps/fw/lib/locker.js

```diff
@@ -92,18 +94,18 @@ function acquireLock(operation, callback) {
 
 
 function acquireSharedLock(callback) {
-    acquireLock(mod_lockfd.LOCK_SH, callback);
+    acquireLock('sh', callback);
```

Comment:
Yuck replacing a constant with a string. So used to enum's in rust for this now :)